ompi/attribute/attribute.c

```diff
@@ -109,6 +113,38 @@
  *            CALL MPI_COMM_GET_ATTR(..., ret, ierr)
  *            --> ret will equal &bar
  *
+ * Cases where C writes an int attribute:
```

Comment:
These 3 cases are not just where C writes an int -- the RMA attributes in question can be `(void*)`, `(MPI_Aint)`, or `(int)`.  Should probably make that clear in the title, the description below, and the text of these 3 cases.